### PR TITLE
Profile page [TAS-54]

### PR DESCRIPTION
**Pull request by femacedogomes**

### CODE REVIEW

**Comment by femacedogomes** [comment]
Neste Pr também foi removido a classe de Dark Theme dos components

File: src/app/(accounts)/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import React from "react";
+import { FC } from "react";
+
+export interface CommonLayoutProps {
+  children?: React.ReactNode;
+}
+
+const pages: {
```
Comment (pedrohbl03) [inline src/app/(accounts)/layout.tsx:13]: u can create a navigation.constants and place this object; 
Because this makes maintenance easy everywhere

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]
Change the comment in accounts/layout plz.

**Comment by femacedogomes** [comment]
just waiting for new end point delete user to really finish. But we can also made a correction in other components.

**Review by renanpontez — APPROVED** [review]

**Pull request merged**